src/components/modals/modals.ts

```diff
@@ -41,7 +42,7 @@ export const modals = {
   [Modals.PromptModal]: PromptModal,
   [Modals.ShareServerModal]: ShareServerModal,
   [Modals.TestCaseModal]: TestCaseModal,
-  [Modals.TestModal]: TestModal,
+  [Modals.ResetProjectModal]: ResetProjectModal,
```

Comment:
```suggestion
  [Modals.TestModal]: TestModal,
  [Modals.ResetProjectModal]: ResetProjectModal,
```
TestModal is still needed 😉